### ISSUE TITLE
Support arguments with spaces.

### ISSUE THREAD

**Issue by kboshold** [microsoft/vscode-gradle#629]
If you want to run a task as "Run task with args" and your argument contains spaces within the value, it will not work. Especially if you want to work with the --args from gradle, this is required.

Example:
Your application required an input and output file and can also use "--debug" to output additional debug information. So you want to run the `gradle run` task with the following arguments:
`--args="input.txt -o output.txt --debug"`

Currently you would get the following output:

```text
> Executing task: gradle: run --args="input.txt -o output.txt --debug" <

<-------------> 0% WAITING

FAILURE: Build failed with an exception.

* What went wrong:
Problem configuring task :run from command line.
> Unknown command-line option '-o'.

* Try:
Run gradle help --task :run to get task usage details. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Could not execute build using Gradle distribution 'https://services.gradle.org/distributions/gradle-6.5.1-bin.zip'.
```

**Comment by badsyntax** [comment]
Thanks for the report, will try get a fix out asap.

**Comment by badsyntax** [comment]
I've released the fix with [3.4.2](https://github.com/badsyntax/vscode-gradle/releases/tag/3.4.2). Can you test it and let me know if you have any issues? Thanks.

**Comment by badsyntax** [comment]
I assume this is fixed, closing.
